cli/command/stack/cmd.go

```diff
@@ -43,13 +42,7 @@ func NewStackCommand(dockerCli command.Cli) *cobra.Command {
 	}
 	defaultHelpFunc := cmd.HelpFunc()
 	cmd.SetHelpFunc(func(cmd *cobra.Command, args []string) {
-		config := cliconfig.LoadDefaultConfigFile(dockerCli.Err()) // dockerCli is not yet initialized, but we only need config file here
```

Comment:
Modified the last commit, and removed the warning, but kept these changes

Reply:
Discussing with @dhiltgen - dropping this commit for now, and keeping it for https://github.com/docker/cli/pull/1139